### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by ElginL**
Add view section to the userguide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -208,6 +209,16 @@ Examples:
 * `find Mac` returns `MacDonald`
 * `find fairprice` returns `NTUC Fairprice` <br>
 
+### Viewing a company : `view`
+
+`view` displays the points of contact and transaction log of a company.
```
Comment (jicsontoh) [inline docs/UserGuide.md]: Perhaps could consider replacing it with "Displays the points of contact and transaction log of the specified company."

**Review by jicsontoh — APPROVED** [review]
LGTM! Descriptive view feature!

File: docs/UserGuide.md (hunk 1 above)
Comment (ElginL) [reply docs/UserGuide.md]: Hi, I've just updated it to what you said

**Review by notle1706 — APPROVED** [review]
LGTM

**Pull request merged**